tsconfig.json

```diff
@@ -6,6 +6,10 @@
     "esModuleInterop": true,
     "outDir": "./dist",
     "rootDir": "./src",
-    "noImplicitAny": true
+    "noImplicitAny": true,
+
+    "paths": {
+      "*": "types/*"
+    }
```

Comment:
Okay so this is is the bit of config I was missing when I tried to organize it this way initially.

It seems a little weird to me that this works (path resolution outside of root dir) but `includes` doesn't let you include stuff outside of the root dir.